source/clear-linux/tutorials/kubernetes.rst

```diff
@@ -99,111 +106,104 @@ deployment and your security needs.
 
          On systems with limited resources, some performance degradation may
          be observed while swap is disabled.
-         
-#. Create (or edit if it exists) the hosts file that kubernetes will read to 
-   locate master's host:
+
+#. Switch to root to modify `hostname`:
 
    .. code-block:: bash
 
-      echo "127.0.0.1 localhost `hostname`" >> /etc/hosts
+      sudo -s
 
-#. Configure the Kubernetes runtime interface, either:
+#.  Create (or edit if it exists) the hosts file that Kubernetes will read to
+    locate the master's host:
 
-   a) Run Kubernetes with Docker + runc:
+    .. code-block:: bash
 
-      #. Enable docker.service:
+       echo "127.0.0.1 localhost `hostname`" >> /etc/hosts
 
-         .. code-block:: bash
+#.  Exit root:
 
-            sudo systemctl enable docker.service
+    .. code-block:: bash
 
-      #. Create (or edit if it exists) the file 
-         :file:`/etc/systemd/system/docker.service.d/51-runtime.conf` and include the following lines:
+       exit
 
-         .. code-block:: bash
+Configure and run Kubernetes
+****************************
 
-            [Service]
-            Environment="DOCKER_DEFAULT_RUNTIME=--default-runtime runc"
+This section describes how to configure and run Kubernetes with:
 
-      #. Create (or edit if it exists) the file :file:`/etc/systemd/system/kubelet.service.d/10-kubeadm.conf` and include the following lines:
+* Docker and runc
+* CRI-O and kata-runtime
 
-         .. code-block:: bash
+Configure and run Docker + runc
+===============================
 
-            [Service]
-            Environment="KUBELET_EXTRA_ARGS="
+#.  Enable the Docker service:
 
-      #. Restart services:
+    .. code-block:: bash
 
-         .. code-block:: bash
+       sudo systemctl enable docker.service
 
-            sudo systemctl daemon-reload
-            sudo systemctl restart docker
-            sudo systemctl restart kubelet
+#.  Create (or edit if it exists) the file 
+    :file:`/etc/systemd/system/docker.service.d/51-runtime.conf` and include the following lines:
 
-   or:
+    .. code-block:: bash
 
-   b) Run Kubernetes with CRI-O + your desired runtime. You can use multiple
-      runtimes with CRI-O, including *runc* and *kata-runtime*. To use
-      CRI-O + *kata-runtime*:
+       [Service]
+       Environment="DOCKER_DEFAULT_RUNTIME=--default-runtime runc"
 
-      #. Enable crio.service:
+#.  Create (or edit if it exists) the file :file:`/etc/systemd/system/kubelet.service.d/10-kubeadm.conf` and include the following lines:
 
-         .. code-block:: bash
+    .. code-block:: bash
 
-            sudo systemctl enable crio.service
+       [Service]
+       Environment="KUBELET_EXTRA_ARGS="
```

Comment:
I believe `KUBELET_NETWORK_ARGS` should also be reset otherwise, the node remains in a non-ready state because it can't find the cni network plugin.

It'd look something like:
```ini
[Service]
Environment="KUBELET_EXTRA_ARGS="
Environment="KUBELET_NETWORK_ARGS="
```